yasfb/formatter.py

```diff
@@ -0,0 +1,623 @@
+# Copyright (c) 2008, Luke Maurits <luke@maurits.id.au>
```

Comment:
I don't mind vendoring the code, but let's include a link to the original upstream source as an acknowledgement.